jsonnet/telemeter/jsonnetfile.json

```diff
@@ -18,7 +18,7 @@
                     "subdir": "jsonnet/prometheus-operator"
                 }
             },
-            "version": "v0.30.0"
+            "version": "release-0.30"
```

Comment:
good catch, but am I assuming correctly that we have to fix this also in the `release-4.1` branch?
https://github.com/openshift/telemeter/blob/c51e426c4ab344364b83cbdc5a76b096c13030b2/jsonnet/telemeter/jsonnetfile.json#L21

Reply:
Yes, we should cherry-pick.

Reply:
it's already there. 4.1 was the first branch to be patched: https://github.com/openshift/telemeter/pull/173

Reply:
the point is that both of these should be pointed at the minor release series, instead of the exact release